examples/purchaseTesterTypescript/app/screens/FooterPaywallScreen.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { PaywallFooterView } from 'react-native-purchases-ui';
+
+import { SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import RootStackParamList from '../RootStackParamList';
+
+type Props = NativeStackScreenProps<RootStackParamList, 'FooterPaywall'>;
+
+const FooterPaywallScreen: React.FC<Props> = () => {
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      justifyContent: 'center',
+      backgroundColor: '#ecf0f1',
+    },
+  });
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <PaywallFooterView>
```

Comment:
kinda wonder whether we should name this a `PaywallFooterContainerView` or something?

Reply:
I'm fine with the current name though

Reply:
I kinda prefer adding the `Container` word there, so it's clearer how it's intended to be used.